scss/civicrm/common/_collapsible-block.scss

```diff
@@ -0,0 +1,31 @@
+.crm-collapsible {
```

Comment:
@mukeshcompucorp i want you to document in the PR, in a "Technical details" section, all this new common partials that you are adding. Document what they are for, show a before/after screenshot. Not of the whole page, but of the specific component that these partials are targeting

Reply:
Ok I will though, this component seems completely un-relevent to this PR. It's because I was working ahead in time of blockage at Christmas. I will remove and add it with appropriate PR with Documentation